dojo/filters.py

```diff
@@ -135,6 +135,20 @@ def __init__(self, *args, **kwargs):
         super(CharFilter, self).__init__(*args, **kwargs)
 
 
+class CharFieldFilterANDExpression(CharFieldInFilter):
+    def filter(self, queryset, value):
+        # Catch the case where a value if not supplied
+        if not value:
+            return queryset
+        # Do the filtering
+        objects = value.split(",")
```

Comment:
```suggestion
        objects = set(value.split(","))
```
Removing duplicate values per feedback from @dogboat